app/services/mixpanel_service.rb

```diff
@@ -121,6 +123,22 @@ def send_event(distinct_id:,
       )
     end
 
+    SECURITY_METRICS_SUBNET = IPAddr.new("192.211.152.0/24").freeze
+    EC2_COMPUTE_DOMAIN_REGEX = /^ec2-(.*)\.compute-1\.amazonaws.com$/i.freeze
+
+    def should_event_be_dropped?(request)
+      return :hund if request.query_parameters[:source]
```

Comment:
it might be better for `should_event_be_dropped?` to return true/false instead of a symbol

also this line seems off -- should probably be `return true if request.query_parameters[:source] == 'hund'`

Reply:
Indeed, not sure how the tests missed that one